sdcm/cluster_k8s/gke.py

```diff
@@ -121,8 +134,9 @@ def setup_gke_cluster(self, num_nodes: int) -> None:
                        f" --local-ssd-count {self.gce_n_local_ssd}"
                        f" --node-taints role=scylla-clusters:NoSchedule"
                        f" --enable-stackdriver-kubernetes"
+                       f"{autoscale_option}"
                        f" --no-enable-autoupgrade"
-                       f" --no-enable-autorepair"
+                       f" --enable-autorepair"
```

Comment:
Do we want to change this parameter in Scylla Operator deployment guide too? I think it's better to stay with `--no-enable-autorepair` for jobs which do not autoscale.

I.e.:

```python
f" {'--enable-autorepair' if autoscale_option else '--no-enable-autorepair'}"
```